QuizWhois.Domain/Services/Implementations/PackService.cs

```diff
@@ -51,6 +52,29 @@ public PackModelResponse GetPack(long packId)
             return entity.ToPackModelResponse();
         }
 
+        public SomePacksModelResponse GetAll(bool? isDraft = null)
+        {
+            List<Pack> entities;
+            if (isDraft != null)
+            {
+                entities = _dbContext.Packs.Where(x => x.IsDraft == isDraft)
+                .Include(x => x.Questions)
```

Comment:
JFYI: if you want to avoid copypasting here, you can create a query with ifs and then add include and evaluate it by calling ToList